cloud/installation/deploy-poller.md

```diff
@@ -17,7 +17,8 @@ Before deploying a poller, make sure the [prerequisites](prerequisites.md) for y
 
    1. Edit the configuration file **/etc/sysconfig/selinux**.
    2. Set the value of SELINUX to disabled (**SELINUX=disabled**).
-   3. Save the file and reboot the system to apply the change.
+   3. By default, the name of the poller is its hostname (this may be shortened) but can be changed using 'hostnamectl set-hostname your-hostname'.
```

Comment:
At this point there is no poller yet so users won't understand the sentence. Also, you put the sentence in the middle of the SELinux instructions.
suggestion:
By default, the hostname of the machine will become the name of the poller. If you want to change the machine's hostname, use the following command: